content/docs/guides/configure-metrics.mdx

```diff
@@ -0,0 +1,129 @@
+---
+# cSpell:ignore XPOST tsdb
+
+title: Configure Metrics in Pomerium Enterprise
+sidebar_label: Configure Metrics
+description: Learn how to configure Prometheus to power metrics in the Pomerium Enterprise web console.
+lang: en-US
+keywords: [pomerium, enterprise pomerium, telemetry, metrics, prometheus]
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to configure [Prometheus](https://prometheus.io/) as a metrics collection backend in Pomerium Enterprise.
+
+## Overview
+
+Pomerium Enterprise uses Prometheus to collect observability and monitoring metrics from your Pomerium deployment. To set up Pomerium to work with Prometheus, you must configure either an external or embedded Prometheus instance.
+
+This guide shows you how to configure both the external and embedded Prometheus options.
+
+## Before you start
+
+To complete this guide, you need:
+
+- [Pomerium Enterprise](/docs/enterprise)
+- [Pomerium Core](/docs/core)
+
+This guide runs both Pomerium instances on localhost (`127.0.0.1`).
+
+## Configure Pomerium
+
+1. In your Pomerium configuration file, define the [`metrics_address`](/docs/reference/metrics#metrics-address) key to a network interface or port. In the example below, Pomerium provides metrics at port `9091` on an IP address reachable by Pomerium Enterprise:
+
+   ```yaml title="pomerium.yaml"
+   metrics_address: 127.0.0.1:9091
+   ```
+
+   :::note
+
+   If you're running Pomerium Enterprise in a distributed environment where the IP address is unknown at the time of deployment, you can use the resolvable fully qualified domain name (FQDN) of the Pomerium host (for example, `pomerium0.internal.example.com`). Or, you can override this key with the [`METRICS_ADDRESS`](/docs/reference/metrics#metrics-address) environment variable.
+
+   We do not recommend exposing this endpoint to public traffic as it can contain potentially sensitive information.
+
+   :::
+
+1. In your Pomerium Enterprise configuration file, define the `metrics_addr` key to a network interface or port. For example:
+
+   ```yaml title="pomerium-enterprise.yaml"
+   metrics_addr: 127.0.0.1:9092
```

Comment:
@wasaga I can't find this setting anywhere, only the `metrics_address` setting. Is this setting still supported? Do we need to add it to the Enterprise configuration page?

Reply:
`metrics_addr` is correct. there's no `metrics_address` setting in the console 

```go
flags.StringVar(&serveOptions.metricsAddr, "metrics-addr", "127.0.0.1:9092", "address to expose console metrics in Prometheus format")
```